.gitignore

```diff
@@ -153,7 +153,7 @@ docs
 
 # Jupyterlab
 **/_build
-**.ipynb
+#**.ipynb Actually, pls send this right now
```

Comment:
I believe there's a reason we don't want ipynb - I'm not ready to undo this until we are sure it's not going to backfire.

Reply:
Reverted .gitignore to exclude ipynb again